src/functions/index.js

```diff
@@ -79,6 +79,22 @@ function separeteTextBy (symbol) {
   }
 }
 
+function agroupElementsAndCount(words) {
```

Comment:
General reducer function that agroup and returns a array which contains the object that have 'element' and 'quantity' keys for each word